content/src/content/jcr_root/apps/core/email/components/segmentation/v1/segmentation/segmentation.html

```diff
@@ -44,19 +41,15 @@
          role="tabpanel"
          aria-labelledby="${item.id}-tab"
          tabindex="0"
-         class="${!wcmmode.disabled ? 'cmp-tabs__tabpanel cmp-tabs__tabpanel--active' : ''}"
-         data-cmp-hook-tabs="tabpanel"
-         data-cmp-data-layer="${item.data.json}"></div>
+         class="${!wcmmode.disabled ? 'cmp-tabs__tabpanel cmp-tabs__tabpanel--active' : ''}"></div>
     <table data-sly-test="${wcmmode.disabled}" role="presentation">
         <sly data-sly-repeat.item="${tabs.items}"
              data-sly-call="${itemTemplate.item @ item = item}"
              id="${item.id}-tabpanel-noedit"
              role="tabpanel"
              aria-labelledby="${item.id}-tab"
              tabindex="0"
-             class="${wcmmode.edit ? 'cmp-tabs__tabpanel cmp-tabs__tabpanel--active' : ''}"
-             data-cmp-hook-tabs="tabpanel"
```

Comment:
this data attribute is required by the panelcontainer.js

Reply:
right! not exactly in this place as the table is only rendered for wcmmode == disabled, but earlier it is required; as well as the `data-cmp-is`. The attributes on the enclosing `div` don't matter though as it is unwrapped in wcmmode == disabled anyway. So I removed the unnecessary conditions there as well and streamlined them in the whole file. See https://github.com/adobe/aem-core-email-components/pull/234/commits/3a9c51fb7b90983511ab68e2d315163b358062a2